clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/ops/servergroup/CloneKubernetesAtomicOperation.groovy

```diff
@@ -47,11 +47,13 @@ class CloneKubernetesAtomicOperation implements AtomicOperation<DeploymentResult
   */
   @Override
   DeploymentResult operate(List priorOutputs) {
-    CloneKubernetesAtomicOperationDescription newDescription = cloneAndOverrideDescription()
+    description.source.namespace = description.source.region
```

Comment:
Should this only override the namespace if it's not explicitly set? Wouldn't that behavior be expected if the calls didn't only originate from orca?